tests/unit/expire.tcl

```diff
@@ -601,4 +601,85 @@ start_server {tags {"expire"}} {
            {del foo}
         }
     } {} {needs:repl}
+
+    test {EXPIRE with NX option on a key with ttl} {
+        r SET foo bar EX 100
+        r EXPIRE foo 200 NX
+    } {0}
```

Comment:
i think all these tests would benefit from adding a check on the result of the TTL to check if the expiration time was modified and if the key exists.
i.e. not just verify the return value of the EXPIRE command.
in that case we wanna use very distinctive expiration times so that we don't have timing issues comparing them.
the above test could be ok (100 and 200 seconds).
so we can assert that the TTL is >50 <= 100
```suggestion
    test {EXPIRE with NX option on a key with ttl} {
        r SET foo bar EX 100
        assert_equal [r EXPIRE foo 200 NX] 0
        assert_range [r TTL foo] 50 100
    }
```